src/components/parkingFineSummary/ParkingFineSummary.tsx

```diff
@@ -0,0 +1,153 @@
+ * - Add dividers
+ * - Add image view to vehicle details
+ */
+
+const ParkingFineSummary = (): React.ReactElement => {
+  const { t } = useTranslation();
+
+  return (
+    <>
+      <p>{t('common:required-fields')}</p>
+
+      <div className="summary-container">
+        <Card className="vehicle-details-container" border>
+          <TextInput
+            id="regNumber"
+            label={t('common:fine-info:reg-number:label')}
+            value={t('common:fine-info:reg-number:placeholder')}
+            readOnly
+          />
+          <TextInput
+            id="regNumber"
+            label={t('parking-fine:vehicle-info:type:label')}
+            value={t('parking-fine:vehicle-info:type:placeholder')}
+            readOnly
+          />
+          <TextInput
+            id="regNumber"
+            label={t('parking-fine:vehicle-info:brand:label')}
+            value={t('parking-fine:vehicle-info:brand:placeholder')}
+            readOnly
+          />
+          <TextInput
+            id="regNumber"
+            label={t('parking-fine:vehicle-info:model:label')}
+            value={t('parking-fine:vehicle-info:model:placeholder')}
+            readOnly
+          />
+          <TextInput
+            id="regNumber"
+            label={t('parking-fine:vehicle-info:color:label')}
+            value={t('parking-fine:vehicle-info:color:placeholder')}
+            readOnly
+          />
+        </Card>
+
+        <TextInput
+          className="info-field"
+          id="fineTimeStamp"
+          label={t('common:fine-info:fine-timestamp:label')}
+          value={t('common:fine-info:fine-timestamp:placeholder')}
+          readOnly
+        />
+        <TextInput
+          className="info-field"
+          id="ref-number"
+          label={t('common:fine-info:ref-number:label')}
+          value={t('common:fine-info:ref-number:placeholder')}
+          readOnly
+        />
+
+        <TextInput
+          className="info-field"
+          id="address"
+          label={t('common:fine-info:address:label')}
+          value={t('common:fine-info:address:placeholder')}
+          readOnly
+        />
+        <TextInput
+          className="info-field"
+          id="additionalDetails"
+          label={t('common:fine-info:additional-details:label')}
+          value={t('common:fine-info:additional-details:placeholder')}
+          readOnly
+        />
+
+        <TextArea
+          className="info-field"
+          id="fine-title"
+          label={t('common:fine-info:fine-title:label')}
+          value={t('common:fine-info:fine-title:placeholder')}
+          readOnly
+        />
+        <TextInput
+          className="info-field"
+          id="additionalDetails"
+          label={t('common:fine-info:fine-details:label')}
+          value={t('common:fine-info:fine-details:placeholder')}
+          readOnly
+        />
+        <TextArea
+          className="info-field"
+          id="fine-title"
```

Comment:
Maybe change _ref-number_ and _fine-title_ to _refNumber_ and _fineTitle_ for consistency? On lines 63, 86 and 100.